.changeset/dry-rules-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
+---
+
+only cache GET/HEAD requests with cacheOnDemandPages
```

Comment:
Can we try to reword this changeset to follow our sematics, e.g. `Fixes ... to ..., Fixes ... which ...`

Some guidance questions, not mandatory to answer all, but helpful.
What does this change do?
How does this change do it?
Why is this change necessary?

Reply:
Made an update in https://github.com/withastro/adapters/pull/127/commits/277fc09d372392109850c29e8c4691df93a1602c, how do you like it?